src/containers/Accounts/AccountAssetTab/AccountAssetTab.tsx

```diff
@@ -5,18 +5,25 @@ import { useNavigate } from 'react-router'
 import { useRouteParams } from '../../shared/routing'
 import { AccountIssuedTokenTable } from '../AccountIssuedTokenTable'
 import { AccountNFTTable } from '../AccountNFTTable/AccountNFTTable'
+import { AccountMPTTable } from '../AccountMPTTable/AccountMPTTable'
 import { ACCOUNT_ROUTE } from '../../App/routes'
 
 // TODO: Add state types or convert to react query
 interface Props {
   account: any
 }
 
-const assetTypes = ['issued', 'nft']
+let assetTypes = ['issued', 'nft']
 
 const AccountAssetTabDisconnected = ({ account }: Props) => {
   const { id: accountId = '', assetType = assetTypes[0] } =
     useRouteParams(ACCOUNT_ROUTE)
+
+  const supportsMPT = ['mpt_sandbox', 'devnet'].includes(
```

Comment:
I would assume this is just for testing?

Reply:
Currently we have an alt server for testing. But assuming if the amendment is activated on mainnet, then this code would need to be removed